deepchecks/vision/vision_data/vision_data.py

```diff
@@ -266,3 +264,91 @@ def __iter__(self):
     def __len__(self):
         """Return the number of batches in the batch loader if it is known, otherwise returns None."""
         return len(self._batch_loader) if hasattr(self._batch_loader, '__len__') else None
+
+    def head(self, num: int = 5):
+        """Show data from a single batch of this VisionData. Works only inside a notebook.
+
+        Parameters
+        ----------
+        num: int, default = 5
+            Number of images to show. Does not show more images than the size of single batch
+        """
+        if not is_notebook():
+            print('head function is supported only inside a notebook', file=sys.stderr)
+            return
+        if num < 1:
```

Comment:
should be < 2 + can also validate its an integer

Reply:
Why? the user can't show 1 image?

Reply:
1 image is OK, I'm with Matan

Reply:
1 is better, so change the text to match it (it says larger 1 is a must)